src/wallet/rpcdump.cpp

```diff
@@ -76,15 +76,16 @@ UniValue importprivkey(const UniValue& params, bool fHelp)
 {
     if (!EnsureWalletIsAvailable(fHelp))
         return NullUniValue;
-    
-    if (fHelp || params.size() < 1 || params.size() > 3)
+
+    if (fHelp || params.size() < 1 || params.size() > 4)
         throw runtime_error(
-            "importprivkey \"bitcoinprivkey\" ( \"label\" rescan )\n"
+            "importprivkey \"bitcoinprivkey\" ( \"label\" rescan timestamp)\n"
             "\nAdds a private key (as returned by dumpprivkey) to your wallet.\n"
             "\nArguments:\n"
             "1. \"bitcoinprivkey\"   (string, required) The private key (see dumpprivkey)\n"
             "2. \"label\"            (string, optional, default=\"\") An optional label\n"
             "3. rescan               (boolean, optional, default=true) Rescan the wallet for transactions\n"
+            "4. timestamp            (numeric, optional, default=0) Rescan starts at timestamp\n"
```

Comment:
I prefer luke-jr's solution provided it's not "bool and time at the same time". It could be always timestamp but 0 can mean no rescan or something like that. rescan=1 meaning "rescan true but with the dwfault timestamp which is not 1" is what seems too unintuitive to me (not having one parameter instead of two) and is what I understood luke was proposing (but please clarify if you were suggesting something else).